snooty.toml

```diff
@@ -17,6 +17,7 @@ toc_landing_pages = [
     "/builders",
     "/get-started",
     "/databases-collections",
+    "/connect",
```

Comment:
S: you can omit this from here, it doesn't need to be a landing page. But since we link to it from the index I suggested adding some basic links.